src/main.rs

```diff
@@ -8,41 +8,49 @@ use serde::Deserialize;
 use bad_words::BAD_WORDS;
 
 // allows for spaces so that 'bum' is not found in 'bumblebee'
-fn contains_bad_word(text_input: &str) -> bool {
+fn contains_bad_word(text_input: &str) -> Vec<String> {
```

Comment:
cool! we might want to rename this function to `find_bad_words`, or something like that.